libparsers/tests/xml/dynamic_lighting.cpp

```diff
@@ -2,45 +2,43 @@
 #include <doctest.h>
 #include <iostream>
 
-using namespace parsers::xml;
-using dynamic_lighting::lighting_info;
-using parsers::errors::error_t;
-using parsers::errors::errors_t;
+using namespace std::string_literals;
 
 TEST_SUITE_BEGIN("libparsers - xml - dynamic_lighting");
 
 TEST_CASE("libparsers - xml - dynamic_lighting - test") {
+	// clang-format off
 	std::string const test_value =
 	    "<?xml version=\"1.0\" encoding=\"utf - 8\"?>"
-	    "<openBVE xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsd="
-	    "\"http://www.w3.org/2001/XMLSchema\">"
-	    "<Brightness>"
-	    "<Time>10.00</Time>"
-	    "<AmbientLight>1, 1, 1</AmbientLight>"
-	    "<DirectionalLight>1, 1, 1</DirectionalLight>"
-	    "<LightDirection>1.0, 1.0, 1.0</LightDirection>"
-	    "<CabLighting>80</CabLighting>"
-	    "</Brightness>"
-	    "<Brightness>"
-	    "<Time>12.00</Time>"
-	    "<AmbientLight>160, 160, 160</AmbientLight>"
-	    "<DirectionalLight>160, 160, 160</DirectionalLight>"
-	    "<LightDirection>-0.5, -0.5, -0.5</LightDirection>"
-	    "<CabLighting>255</CabLighting>"
-	    "</Brightness>"
+	    "<openBVE xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd="
+	    "	\"http://www.w3.org/2001/XMLSchema\">"
+	    "	<Brightness>"
+	    "		<Time>10.00</Time>"
+	    "		<AmbientLight>1, 1, 1</AmbientLight>"
+	    "		<DirectionalLight>1, 1, 1</DirectionalLight>"
+	    "		<LightDirection>1.0, 1.0, 1.0</LightDirection>"
+	    "		<CabLighting>80</CabLighting>"
+	    "	</Brightness>"
+	    "	<Brightness>"
+	    "		<Time>12.00</Time>"
+	    "		<AmbientLight>160, 160, 160</AmbientLight>"
+	    "		<DirectionalLight>160, 160, 160</DirectionalLight>"
+	    "		<LightDirection>-0.5, -0.5, -0.5</LightDirection>"
+	    "		<CabLighting>255</CabLighting>"
+	    "	</Brightness>"
 	    "</openBVE>";
+	// clang-format on
```

Comment:
With this style of indentation (Within the quotes) you should be able to leave clang-format on, otherwise you get weird indentations.